lib/tasks/queue_faxes.rake

```diff
@@ -1,4 +0,0 @@
-desc "Queues faxes for signed, unfaxed applicants"
-task queue_faxes: :environment do
-  ExportFactory.export_unfaxed_snap_applications
```

Comment:
remove this method too?

Reply:
I wasn't sure if we should keep it. Let's burn it down 🔥 🔥 🔥 (done in b976e37)

Reply:
Whoops! Yep, I neglected to save and commit that change. Done in 8206ea1